components/Card.tsx

```diff
@@ -0,0 +1,14 @@
+import next from "next";
+import React from "react";
+
+
+
+export default function Card (props:any) {
```

Comment:
- Use arrow functions to define components
- Define a ```type``` for your ```props```. DO NOT use ```any```
```js
type Props = {
      // Define properties
};

const Card: React.FC<Props> = (props) => {
              // Component Code
}

export default Card;
```

Reply:
Error fixed .Card component update